src/probabilities_estimators/permutation_ordinal/spatial_permutation.jl

```diff
@@ -171,3 +200,5 @@ function Base.show(io::IO, est::SpatialSymbolicPermutation{D}) where {D}
     print(io, "\n")
     show(io, MIME"text/plain"(), est.stencil)
 end
+
+alphabet_length(est::SpatialSymbolicPermutation)::Int = factorial(est.m)
```

Comment:
For me this is a fine way to implement `alphabet_length`. cc @kahaaga . we can merge this and you can update #126 ?

Reply:
I think it's fine to define `alphabet_length` like this, @ikottlarz. Just see my comment above on naming `get_m`.